data/test/vtgate/select_cases.txt

```diff
@@ -197,9 +197,84 @@
   }
 }
 
-# invalid keyspace for '*' expression for simple route
-"select a.user.* from user"
-"cannot resolve a.user.* to keyspace user"
+# select * from authoritative table
+"select * from authoritative"
+{
+  "Original": "select * from authoritative",
+  "Instructions": {
+    "Opcode": "SelectScatter",
+    "Keyspace": {
+      "Name": "user",
+      "Sharded": true
+    },
+    "Query": "select authoritative.user_id as user_id, authoritative.col1 as col1, authoritative.col2 as col2 from authoritative",
```

Comment:
For these common cases where there is only one table in the query, I think we should omit the qualifier from each column name.

This is of course correct as-is but I think the unnecessary expansion adds bloat to things like query logs or error messages and as such makes debugging more complicated.